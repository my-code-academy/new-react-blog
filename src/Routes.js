// import React from 'react';
// import { Route, BrowserRouter as Router } from 'react-router-dom';
// import App from './Components/App/App';
// import CreateBlogPost from './Components/CreateBlogPost/CreateBlogPost';

// const routing = (
//   <Router>
//     <div>
//       <Route exact path="/" component={App} />
//       <Route exact path="/create" component={CreateBlogPost}/>
//     </div>
//   </Router>
// )

import App from './Components/App/App';
import CreateBlogPost from './Components/CreateBlogPost/CreateBlogPost';
import { Route, BrowserRouter as Router } from 'react-router-dom';
import React from 'react';
import CreateBlogPostContainer from './Containers/CreateBlogPostContainer'

const Routing = () => {
  return (
    <Router>
      <div>
        <Route exact path="/" component={App} />
        <Route path="/create" component={CreateBlogPostContainer} />
      </div>
    </Router>
  );
}

export default Routing;