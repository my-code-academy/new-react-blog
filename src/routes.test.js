import React from 'react';
import { shallow } from 'enzyme';
import Routes from './routes';
import renderer from 'react-test-renderer';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });

describe('MyComponent', () => {
  it("should render properly", () => {
    const tree = renderer.create(Routes).toJSON();
    expect(tree).toMatchSnapshot();
  });
});