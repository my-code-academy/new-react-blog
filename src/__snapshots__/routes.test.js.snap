// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent should render properly 1`] = `null`;
